libraries/adaptive-expressions/src/builtinFunctions/sentenceCase.ts

```diff
@@ -25,12 +27,17 @@ export class SentenceCase extends StringTransformEvaluator {
     /**
      * @private
      */
-    private static evaluator(args: any[]): string {
-        const inputStr = String(InternalFunctionUtils.parseStringOrUndefined(args[0])).toLowerCase();
+    private static evaluator(args: any[], options: Options): ValueWithError {
+        let locale = options.locale ? options.locale : Intl.DateTimeFormat().resolvedOptions().locale;
+        locale = FunctionUtils.determineLocale(args, 2, locale);
+        const inputStr = (InternalFunctionUtils.parseStringOrUndefined(args[0]) as any).toLocaleLowerCase(locale);
```

Comment:
Do we need the `as` casting? Why doesn't `InternalFunctionUtils.parseStringOrUndefined` return a well-typed result?

Reply:
InternalFunctionUtils.parseStringOrUndefined returns a string.
However while runing `tsc -b`, it throws 
InternalFunctionUtils.parseStringOrUndefined(args[0]).toLocaleLowerCase(locale)
```
error TS2554: Expected 0 arguments, but got 1.
```